### PR TITLE
Move configuring workspace exposure strategies page into installation guide

### PR DESCRIPTION
**Pull request by mshaposhnik**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Move configuring workspace exposure strategies page into inslattation guide & minor procedure fixups

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/17271

### Specify the version of the product this PR applies to.
7.19

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [x] Dashboard [branding.constant.ts](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.constant.ts) + [product.json](https://github.com/eclipse/che-dashboard/blob/master/src/assets/branding/product.json)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/installation-guide/pages/configuring-workspace-exposure-strategies.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [id="configuring-workspace-exposure-strategies"]
 // = Configuring workspace exposure strategies
 :navtitle: Configuring workspace exposure strategies
-:keywords: end-user-guide, configuring-workspace-exposure-strategies
+:keywords: installation-guide, configuring-workspace-exposure-strategies
 :page-aliases: .:configuring-workspace-exposure-strategies
```
Comment (themr0c) [inline modules/installation-guide/pages/configuring-workspace-exposure-strategies.adoc]: Please add also a redirect:

```suggestion
:page-aliases: .:configuring-workspace-exposure-strategies, end-user-guide:configuring-workspace-exposure-strategies
```

File: modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-an-operator.adoc [hunk 2]
```diff
@@ -16,18 +16,28 @@ link:https://docs.openshift.com/container-platform/latest/operators/olm-what-ope
...
+    ingressStrategy: '__single-host__'
+----
+
+
 * Update the YAML file properties of an already running cluster by executing the `{orch-cli} patch` command. For example:
 +
 [subs="+quotes,+attributes"]
 ----
-$ {orch-cli} patch checluster {prod-checluster} --type=json -p '[{"op": "replace", "path": "/spec/k8s/ingressStrategy", "value": "__single-host__"}]'
+$ {orch-cli} patch checluster {prod-checluster} --type=json -p '[{"op": "replace", "path": "/spec/k8s/ingressStrategy", "value": "__single-host__"}] -n <namespace-name>'
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-an-operator.adoc]:
```suggestion
$ {orch-cli} patch checluster {prod-checluster} --type=json -p '[{"op": "replace", "path": "/spec/k8s/ingressStrategy", "value": "__single-host__"}] -n __<namespace-name>__'
```

File: modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-an-operator.adoc [hunk 3]
```diff
@@ -16,18 +16,28 @@ link:https://docs.openshift.com/container-platform/latest/operators/olm-what-ope
 When deploying {prod-short} using the Operator, configure the intended strategy by modifying the `spec.k8s.ingressStrategy` property of the CheCluster Custom Resource object YAML file.
 To activate changes done to CheCluster YAML file, do one of the following:
 
-* Create a new cluster by executing the `{orch-cli} apply` command. For example:
+* Create a new cluster by executing the `{prod-cli}` command with applying patch. For example:
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-an-operator.adoc]:
```suggestion
* Create a new cluster by executing the `{prod-cli}` command with applying a patch. For example:
```

**Review by rkratky — COMMENTED** [review]
Thanks, @mshaposhnik. I added a couple of suggestions. Also, could you please check the 'PR checklist' in the PR description? Lastly, could you please squash your commits when done modifying the PR?

**Comment by mshaposhnik** [comment]
@rkratky I have checked, there is no direct links to this page.

**Comment by mshaposhnik** [comment]
@rkratky is it ok to merge now?

**Comment by themr0c** [comment]
https://github.com/eclipse/che-docs/pull/1530/files is moving the same doc to another guide: the Administration Guide. Which PR should have precedence?

**Comment by mshaposhnik** [comment]
We agreed with @tomgeorge that we will merge my PR. So @rkratky @themr0c if looks good for you, pls set final approve since i have to continue work on documenting new single-host flavour.

**Review by themr0c — APPROVED** [review]

**Pull request merged**